Convergence/IO/EPICS/CA/ChannelAccessWrapper.cs

```diff
@@ -82,15 +78,15 @@ static ChannelAccessDLLWrapper()
         // CONTEXT
         // -------
 
-        public static IntPtr CurrentContext { get; private set; }
+        public static nint CurrentContext { get; private set; }
 
         private static void set_current_context()
         {
             var context = ca_current_context();
             // Check if CurrentContext has been created
-            if (CurrentContext == IntPtr.Zero)
+            if (CurrentContext == nint.Zero)

```

Comment:
I liked that you are using modern C# datatypes 😃

Reply:
Actually... not very consistently ... maybe I will just replace all IntPtr with nint